git/objects/commit.py

```diff
@@ -752,11 +752,11 @@ def co_authors(self) -> List[Actor]:
 
         if self.message:
             results = re.findall(
-                r"^Co-authored-by: ((?:\w|\-| ){0,38}) <(\S*)>$",
+                r"^Co-authored-by: ((?:\w|\-| ){0,38} <\S*>)$",
```

Comment:
To clarify, can you explain where this regex is coming from? Earlier I tried to argue that the one used by `Actor` could be used here. It's much more open than this, but I can be swayed if there is a specification of the `Co-authored-by:` field that limits it.

Reply:
Ah, my bad! Regarding the name part, this is on me, I used a regex to match against the GitHub username specs, but this is just the account name, so a more generic one is necessary - switched to match the regex used in `Actor` (with the extra `Co-authored-by: ` text and start/end line checks).

Also added a line to ignore the flake8 check - not used flake8 but I think this should be right.